packages/client/src/BrubeckNode.ts

```diff
@@ -38,12 +39,13 @@ export default class BrubeckNode implements Context {
     debug
     private startNodeCalled = false
     private startNodeComplete = false
+    private trackers: SmartContractRecord[] = []
```

Comment:
If we change/remove `getTrackerList()` (see below), we can remove this field as it is then only used in `initNode()`.